core/libraries/line_item_display/EE_SPCO_Line_Item_Display_Strategy.strategy.php

```diff
@@ -274,14 +274,24 @@ private function _ticket_row(EE_Line_Item $line_item, $options = array())
             'item_l'
         );
         // price td
-        $html .= EEH_HTML::td($line_item->unit_price_no_code(), '', 'item_c jst-rght');
+        $price = apply_filters(
+            'FHEE__EE_SPCO_Line_Item_Display_Strategy___ticket_row__price',
+            $line_item->unit_price_no_code(),
+            $line_item
+        );
+        $html .= EEH_HTML::td($price, '', 'item_c jst-rght');
         // quantity td
         $html .= EEH_HTML::td($line_item->quantity(), '', 'item_l jst-rght');
         $this->_total_items += $line_item->quantity();
         // determine total for line item
         $total = $line_item->total();
         $this->_events[ $options['event_id'] ] += $total;
         // total td
```

Comment:
the above line is used for calculating a running total for the current line items
the new filter should probably go before it in case it returns something different,
otherwise the running total will be inaccurate 

ie: do something like this instead:

```php
        // determine total for line item
        // $total = $line_item->total();  <<< delete this line and just put the call to total() inside the filter
        $total = apply_filters(
            'FHEE__EE_SPCO_Line_Item_Display_Strategy___ticket_row__total',
            $line_item->total(),
            $line_item
        );
        $this->_events[ $options['event_id'] ] += $total;
```